### ISSUE TITLE
The route "pimcore_admin_asset_getimagethumbnail" does not exist

### ISSUE THREAD

**Issue by mc-supply** [pimcore/pimcore#6533]
# Bug Report

Since updating to 6.6.5, images using image editable no longer display in editmode.  Instead, an error 'Uncaught Error: The route to "pimcore_admin_asset_getimagethumbnail" does not exist.' is thrown in the Javascript console.  This even appears on demo.pimcore.fun.

Steps to reproduce: Add any image editable and add an image.

![image](https://user-images.githubusercontent.com/54868988/83819102-40ba4880-a686-11ea-9067-8dd96837dc7e.png)

![image](https://user-images.githubusercontent.com/54868988/83817917-2e8adb00-a683-11ea-9a73-1857d7fbb355.png)

**Comment by jdreesen** [comment]
That's strange, because the route is defined here: 
https://github.com/pimcore/pimcore/blob/v6.6.5/bundles/AdminBundle/Controller/Admin/AssetController.php#L1251

**Comment by mc-supply** [comment]
Not entirely sure?  It occurs both when adding a new image as well as when displaying existing content within the admin interface.

**Comment by tevesh** [comment]
Hi there,
i've encountered the same issue with complex template.
The fix for now is to inject the following lines of code into project's main template (file **.html.php**).
Otherwise i'm not be able to correctly save the page during editing inside backend.
```php
<?php if ($this->editmode) { ?>
        <script src="<?php echo $view->router()->path('fos_js_routing_js', array('callback' => 'fos.Router.setData')) ?>"></script>
<?php
} ?>
```

**Comment by dvesh3** [comment]
regression of https://github.com/pimcore/pimcore/pull/6146

**Comment by dvesh3** [comment]
@mc-supply can you please confirm if https://github.com/pimcore/pimcore/pull/6534 fixes your issue? thanks!

**Comment by mc-supply** [comment]
@dvesh3, I can confirm that fixes it.  Thank you for the quick reply!

**Comment by weisswurstkanone** [comment]
PR needs to be merged first

**Comment by mc-supply** [comment]
Sounds good.  Thank you for reopening it.

**Comment by dvesh3** [comment]
resolved by #6534

**Comment by alex-jungwirth** [comment]
Is there a workaround for twig atm?
